source/Camera/Basic/CaptureWithSettingsFromYML/CaptureWithSettingsFromYML.cpp

```diff
@@ -1,7 +1,7 @@
 /*
 This example shows how to capture point clouds, with color, from the Zivid camera,
-with settings from YML file. The YML files for this samples can be found at:
-https://zivid.atlassian.net/wiki/spaces/ZividKB/pages/450363393/Sample+Data
+with settings from YML file. The YML files for this samples can be found under the main instructions
```

Comment:
Can you apply this change to the other samples that use sample data?
Hint: Open the repo folder in Visual Studio Code and search for ZIVID_SAMPLE_DATA_DIR - all samples should be listed

Reply:
@ajlandau have you seen this comment?

Reply:
Also, there is a typo -> The YML files for this sample**s** - should be singular

Reply:
Added the last sentence to all of the files.  fixed plural to singular. For the files that use other data in the sample data folder, changed what they were finding.
yml -> zdf
yml -> yaml  
to make it valid for the specific sample, is that okay?